### PR TITLE
[Feature] Unify flink configuration

### LINKED ISSUE #1857: [Proposal] Unify flink configuration
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/streampark/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Currently, there is no unified specification for parameter settings in the streampark project, so this time we will solve the problem of the specification of the whole parameter setting It involves setting the env environment of the job (stream env | table env), user's parameters, and user's flinksql content

before:
```
flink:
  deployment:
    property:
       ${StreamExecutionEnvironment.key} : $value
  # table
  table:
    planner: blink # (blink|old|any)
    mode: streaming #(batch|streaming)
```

after:
```
env:
  option: #cli opiton args
    target: yarn-application # yarn-application, yarn-perjob
    shutdownOnAttachedExit:
    jobmanager:
    ...
  property: 
    ${StreamExecutionEnvironment.key} : $value
    ...
    table: 
      ${TableEnvironment.key} : $value
      ...
sql: # flinksql
   my_flinksql: |
    CREATE TABLE datagen (
      f_sequence INT,
      ts AS localtimestamp,
      WATERMARK FOR ts AS ts
    ) WITH (
      ....
    );
    ...
```

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by wolfboys**
Currently, there is no unified specification for parameter settings in the streampark project, so this time we will solve the problem of the specification of the whole parameter setting It involves setting the env environment of the job, table' parameters, user's parameters, and user's flinksql content

before:
```
flink:
  deployment:
    property:
       ${StreamExecutionEnvironment.key} : $value
  # table
  table:
    planner: blink # (blink|old|any)
    mode: streaming #(batch|streaming)
```

after:
```
flink:
  option: #cli opiton args
    target: yarn-application # yarn-application, yarn-perjob
    shutdownOnAttachedExit:
    jobmanager:
    ...
  property: 
    ${StreamExecutionEnvironment.key} : $value
    ...
  table: 
    ${TableEnvironment.key} : $value
     ...

app: # user's parameter
  # $key: $value

sql: # flinksql
   my_flinksql: |
    CREATE TABLE datagen (
      f_sequence INT,
      ts AS localtimestamp,
      WATERMARK FOR ts AS ts
    ) WITH (
      ....
    );
    ...
```

## Contribution Checklist

  - If this is your first time, please read our contributor guidelines: [Submit Code](https://streampark.apache.org/community/submit_guide/submit_code).

  - Make sure that the pull request corresponds to a [GITHUB issue](https://github.com/apache/streampark/issues).

  - Name the pull request in the form "[Feature] Title of the pull request", where *Feature* can be replaced by `Hotfix`, `Bug`, etc.

  - Fill out the template below to describe the changes contributed by the pull request. That will give reviewers the context they need to do the review.

  - If the PR is unfinished, add `[WIP]` in your PR title, e.g., `[WIP][Feature] Title of the pull request`.

-->

## What changes were proposed in this pull request

Issue Number: close #1857 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)

### CODE REVIEW

**Comment by wolfboys** [comment]
cc @1996fanrui PTAL

File: streampark-console/streampark-console-service/src/main/resources/flink-application.conf [hunk 1]
```diff
@@ -15,60 +15,61 @@
...
-        attempts: 3
-        delay: 5000
-      restart-strategy.failure-rate:
-        max-failures-per-interval:
-        failure-rate-interval:
-        delay:
-  # table
-  table:
-    planner: blink # (blink|old|any)
-    mode: streaming #(batch|streaming)
```
Comment (1996fanrui) [inline streampark-console/streampark-console-service/src/main/resources/flink-application.conf]: Hi @wolfboys , thanks for your contribution.

Why delete table related configs?

File: streampark-common/src/main/scala/org/apache/streampark/common/util/FileUtils.scala [hunk 2]
```diff
@@ -23,6 +23,104 @@ import scala.collection.JavaConversions._
...
+      val hv: String = Integer.toHexString(v).toUpperCase
+      if (hv.length < 2) {
+        stringBuilder.append(0)
+      }
+      stringBuilder.append(hv)
+    }
+    stringBuilder.toString
+  }
+
+  def getFileType(file: File): String = {
```
Comment (1996fanrui) [inline streampark-common/src/main/scala/org/apache/streampark/common/util/FileUtils.scala:109]: Why change this file? I didn't see anywhere call this method.

File: streampark-console/streampark-console-service/src/main/resources/flink-application.conf (hunk 1 above)
Comment (wolfboys) [reply streampark-console/streampark-console-service/src/main/resources/flink-application.conf]: > Hi @wolfboys , thanks for your contribution.
> 
> Why delete table related configs?

oh, These are not standard parameters in Flink, These parameters can be set in `env.property `

**Comment by wolfboys** [comment]
hi 1996fanrui: 

Regarding the rationality and role of the parameter settings in this part, since many developers rarely use the API provided in streampark to develop flink jobs, it is difficult to understand the role and impact of this part, and it is difficult to fully communicate, but I As the original designer, I have reasons to change this part. This pr has been proposed for a long time and can be merged first.

**Comment by MonsterChenzhuo** [comment]
LGTM

**Review by MonsterChenzhuo — APPROVED** [review]

**Pull request merged**